1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_2_Palindrome/src/index.spec.ts

```diff
@@ -1,4 +1,27 @@
+import palindrome from "./index";
 
-describe('palindrome checker', () => {
+describe("palindrome checker", () => {
+  it("'mom' should be palindrome", () => {
+    expect(palindrome("mom")).toBeTruthy();
+  });
 
-})
\ No newline at end of file
+  it("'wow' should be palindrome", () => {
```

Comment:
Hello! Looks good overall. 👍🏻 

Does this test case bring any value? The first test case seems very similar to this one. I think that if you remove this test then it does not affect overall confidence. The same argument could be applied to the last and before the last test cases.

What do you think?

Also, you check complex phrases if they are a palindrome but you don't check other complex phrases if they are not a palindrome.

Reply:
Yes that is true, bit confused here what to do.
Asked a question here about extra examples https://discord.com/channels/997940575590416435/1086572775013097493

Reply:
Hello guys !
What I would do here is I would use a false example.
e.g:
```
experct(palindrome("tony")).toBeFalse();
```

This would force me to go to the next obvious implementation.